tests/mock_network/churn.rs

```diff
@@ -234,9 +237,37 @@ fn random_churn<R: Rng>(
     assert!(max_drop > 0);
     let dropped_nodes = drop_random_nodes(rng, nodes, Some(max_drop));
     warn!("Dropping nodes: {:?}", dropped_nodes);
+
     BTreeSet::new()
 }
 
+// To ensure the dropped nodes are removed from correspondent section members.
+// Sending out enough user data so that the dropped node can be detected via unresponsive tracking.
+fn verify_dropped_nodes<R: Rng>(
+    rng: &mut R,
+    mut nodes: &mut Vec<TestNode>,
+    dropped_nodes: &BTreeSet<PublicId>,
+) {
+    let is_still_elder = |node: &TestNode| {
+        dropped_nodes
+            .iter()
+            .any(|dropped_node| node.inner.is_peer_our_elder(dropped_node))
```

Comment:
This is probably fine currently, but once promotion/demotion is implemented, we might in theory have a situation where a node that is supposed to be removed is only demoted and this test would still pass. Calling `is_peer_our_member` instead might make the tests more robust.

Reply:
yeah, good suggestion, I will change to use `is_peer_our_member`